modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -32,7 +33,139 @@ With those steps completed, stop your webserver.
+
+.  The ownCloud server data directory
+.  The complete ownCloud directory
+.  The ownCloud server database
+
+You can find detailed steps to do backups here: xref:maintenance/backup.adoc[].
+
+[source,console]
+----
+
+# This example assumes Ubuntu Linux and MariaDB
+mv /var/www/owncloud /var/www/owncloud-old-version-number
+mysqldump -u<username> -p<password> <databasename> > <ownCloud-Version-Dump.sql>
+----
+
+== Review Third-Party Apps
+
+# Enable maintenance mode using the occ command.
+sudo -u www-data php occ maintenance:mode --on
+----
+
+In clustered environment please check that all nodes are in maintenance mode.
+== Stop the Webserver
+
+
+Review any installed third-party apps for compatibility with the new ownCloud release.
+Ensure that they are all disabled before beginning the upgrade.
+Third party apps are all apps that are not distributed by https://marketplace.owncloud.com/publishers/owncloud
+
+Then, disable the apps, via the command-line.
+ 
+[source,console]
+----
+# This command lists all apps by <app-id> and app version
+sudo -u www-data php occ app:list
+
+# This command disables the app with the given <app-id>
+sudo -u www-data php occ app:disable <app-id>
+----
+
+== Download the Latest Installation
+
+Download the latest ownCloud server release from https://owncloud.org/install[owncloud.org/install] into an empty directory *outside* of your current installation.
+[source, console]
+----
+wget https://download.owncloud.org/community/owncloud-10.1.1.tar.bz2
+----
+
+NOTE: Enterprise users must download their new ownCloud archives from their accounts on https://customer.owncloud.com/owncloud/.
+
+To download the tarball from https://customer.owncloud.com you will need a command like this:
+
+[source,console]
+----
+wget https://username:password@customer.owncloud.com/link-to-tarball
+----
+
+== Upgrade
+
+NOTE: For this description we assume that your existing ownCloud installation is located in the default location: `/var/www/owncloud`. The path might differ, depending on your installation.
+
+=== Extract the New Source
+
+We described how to backup your existing ownCloud installation at the beginning of this document. You will find it in `/var/www/owncloud-old-version-number`.
+
+Extract the unpacked ownCloud server directory and its contents to the location of your original ownCloud installation.
+
+[source,console]
+----
+tar -C /var/www/ -xvf owncloud-10.1.1.tar.bz2
+----
+
+With the new source files now in place of the old ones, next copy the `config.php` file from your old ownCloud directory to your new ownCloud directory:
+
+[source,console]
+----
+sudo cp /var/www/owncloud-old/config/config.php /var/www/owncloud/config/config.php
+----
+
+If you keep your `data/` directory _inside_ your `owncloud/` directory, move it from your old version of ownCloud to your new version:
+
+[source,console]
+----
+mv /var/www/owncloud-old-version-number/data /var/www/owncloud/data
```

Comment:
IMO we should be consistent. If we assume the backup is in "/var/www/owncloud-old" then we should use this path for the backup throughout the document. Changing back and forth confuses the reader.

Reply:
correct fixed